### PR TITLE
uuf debugger profiler view added

### PR DESCRIPTION
**Pull request by thusithak**
For issue #175. Please review and merge

### CODE REVIEW

File: components/uuf-core/src/main/resources/apps/debug/public/js/profiler.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+$( document ).ready(function() {
+
+    var dataUrl = "https://api.myjson.com/bins/b2r97";
```
Comment (thusithak) [inline components/uuf-core/src/main/resources/apps/debug/public/js/profiler.js]: @Kishanthan @this Change `dataUrl` to required end-point

**Comment by thusithak** [comment]
1. Bars that represent fragments are now aligned with group label
2. Tooltip now gets destroyed on mouse-out, flickering defect is also fixed
3. JS code formatted

**Comment by Kishanthan** [comment]
@msmshariq need to add the required json to the PR #107 and then merge this with changing to the correct URL

**Comment by this** [comment]
Closing as repo is [deprecated](https://github.com/wso2/carbon-uuf/pull/278).

**Pull request closed without merging**